src/scripts/main.js

```diff
@@ -1,3 +1,227 @@
 'use strict';
 
-// write code here
+function updateSortDirection(columnIndex) {
+  if (columnIndex === lastSortedColumn) {
+    sortDirection = -sortDirection;
+  } else {
+    sortDirection = 1;
+  }
+
+  lastSortedColumn = columnIndex;
+}
+
+function getCellValue(row, columnIndex) {
+  return currencyToNumber(row.cells[columnIndex].innerText);
+}
+
+function currencyToNumber(value) {
+  if (value[0] === '$') {
+    return +value.slice(1).split(',').join('');
```

Comment:
The `currencyToNumber` function assumes that the currency symbol is always '$'. If there are other currency symbols, this function will not handle them correctly. Consider making this function more flexible or ensuring that input values are always in the expected format.